python/res/job_queue/ext_job.py

```diff
@@ -202,6 +203,9 @@ def get_license_path(self):
     def get_arglist(self):
         return self._get_arglist()
 
+    def get_argvalues(self):
+        return self._get_argvalues()
```

Comment:
should we make this a list and not have a stringList moving around in the python layer?

Reply:
I would second that, I did it to be consistent with `get_arglist`, but perhaps I can change both instead?

Reply:
I would say change both :+1:

Reply:
I just wrapped it in `list` (and added a small comment as to why..)